packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -311,6 +325,13 @@ export default class Jobs {
           true
         )
 
+        type === 'backup' &&
+          app.emit('xo:postCall', {
+            ...data,
+            method: 'backupNg.runJob',
```

Comment:
IMHO, it's more an event than an API call. 

Note that there is an `audit` plugin which records API calls. If we considerate this event as an API call, the XO admin will think that this job is triggered manually.

Reply:
There is no trivial solutions to respond to this problematic.